### PR TITLE
add: terra-mesa

### LINKED ISSUE #1122: [Request] mesa-terra
**What software are you requesting to be packaged?**
https://src.fedoraproject.org/rpms/mesa/blob/rawhide/f/mesa.spec

**Describe the motivation**
Right now RPMFusion only builds mesa-*-drivers-freeworld which rely on vanilla mesa, and causing update issues because of inconsistent update schedules.

We should rebuild all of Mesa with freeworld opts enabled so Terra gets all of Mesa and updates become consistent.

**Additional context**
Add any other context about the package request here.

### PR DESCRIPTION
**Pull request by korewaChino**
This pull request adds the terra-mesa package to the project.

### CODE REVIEW

**Comment by korewaChino** [comment]
Please don't merge right away.

Test packages first and check conflict with other packages relying on normal mesa!!

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/1542)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/1543)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/9991847588) for details

**Pull request merged**
